rest_framework_nested/viewsets.py

```diff
@@ -39,35 +48,35 @@ def _get_parent_lookup_kwargs(self) -> dict:
 
         return parent_lookup_kwargs
 
-    def get_queryset(self):
+    def get_queryset(self) -> QuerySet[T_Model]:
         """
         Filter the `QuerySet` based on its parents as defined in the
         `serializer_class.parent_lookup_kwargs` or `viewset.parent_lookup_kwargs`
         """
-        queryset = super().get_queryset()
+        queryset = super().get_queryset()  # type: ignore[misc]
 
         if getattr(self, 'swagger_fake_view', False):
             return queryset
 
-        orm_filters = {}
+        orm_filters: dict[str, Any] = {}
         parent_lookup_kwargs = self._get_parent_lookup_kwargs()
         for query_param, field_name in parent_lookup_kwargs.items():
-            orm_filters[field_name] = self.kwargs[query_param]
+            orm_filters[field_name] = self.kwargs[query_param]  # type: ignore[attr-defined]
         return queryset.filter(**orm_filters)
 
-    def initialize_request(self, request, *args, **kwargs):
+    def initialize_request(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Request:
         """
         Adds the parent params from URL inside the children data available
         """
-        request = super().initialize_request(request, *args, **kwargs)
-        
+        drf_request: Request = super().initialize_request(request, *args, **kwargs)  # type: ignore[misc]
```

Comment:
I needed to rename this variable, unfortunately, because it clashes with parameter name `request: HttpRequest`.

Using `cast()`, mypy allows reassigning a variable with a different type, but not when using an explicit variable hint `request: Request = ...`